src/components/MarkdownPlugins/image.js

```diff
@@ -7,9 +7,9 @@ const image = {
     image: match[2],
     alt: match[1],
   },
-  toBlock: data => `![${ data.alt || '' }](${ data.image || '' })`,
```

Comment:
@Dammmien I'm just wondering why we want it to be `undefined` instead of an empty string, since that was just implemented recently. I'm not saying it's wrong, I'd just like a little background on why you can to change it.

Reply:
I think his reasoning lies in comparison to the youtube component: